### ISSUE TITLE
server ok?

### ISSUE THREAD

**Issue by diatomsRcool** [jhpoelen/effechecka#68]
Can someone check the effechecka server? I have a list that is taking longer than normal. Just want to make sure I haven't killed it.

**Comment by jhpoelen** [comment]
@diatomsRcool as far as I can tell, the server is alive and well . . . however, there's no resources left (all 192 cpus), so your jobs are getting rejected. Perhaps best to ask @mjcollin or @godfoder - as they are the ones managing the cluster that effechecka runs on.

**Comment by diatomsRcool** [comment]
Hmmmm

So who is gobbling up the resources ;)

**Comment by jhpoelen** [comment]
Must be frustrating to not see how busy the cluster is and not be able to take action. I've created a separate issue here - https://github.com/bio-guoda/guoda-services/issues/19 . 

About the resource usage - there is a freshdata job that is still running and I think I see a bunch of possibly abandoned jupyter notebooks that are still taking up resources (see related https://github.com/bio-guoda/guoda-services/issues/14 ). I've attempted to stop the fresh data job, and do not have enough permissions to stop it completely.
